app/views/includes/help_details.html

```diff
@@ -2,8 +2,10 @@
     <details class="help-panel help-details">
         <summary><span class="summary help-details-summary">{{ common.helpTitle }}</span></summary>
         <div class="panel panel-border-narrow">
-            <p>{{ common.helpText | safe }}</p>
-            <p>{{ common.contactTelLabel | replace('{helpLineNumber}', helpline.number) }} {{ common.contactOpeningTimes | replace('{openingTimes}', helpline.hours) }}</p>
+            <h4 class="heading-small">{{common.helpHeading1 | safe}}</h4>
+            <p>{{ helpline.number | safe }}</p>
+            <p>{{ common.contactOpeningTimes | replace('{openingTimes}', helpline.hours) }}</p>
+            <h4 class="heading-small">{{common.helpHeading2 | safe}}</p>
             <p>{{ common.helpEmailLabel }} {{ common.contactEmailAddress | safe }}</p>
```

Comment:
If you do the change suggested above, then this would change to:
```
<p>{{ common.helpEmailLabel | replace('{contactEmailAddress}', links.contactEmailAddress) | safe }}</p>
```